cliconfig/config.go

```diff
@@ -57,6 +57,7 @@ type AuthConfig struct {
 type ConfigFile struct {
 	AuthConfigs map[string]AuthConfig `json:"auths"`
 	HttpHeaders map[string]string     `json:"HttpHeaders,omitempty"`
+	PsFormat    string                `json:"psFormat,omitempty"`
```

Comment:
never mind - I see now that this is part of the config file - so this makes sense!